modules/swagger-jaxrs2/src/test/java/io/swagger/v3/jaxrs2/resources/jspecify/nullunmarked/package-info.java

```diff
@@ -0,0 +1,4 @@
+@NullUnmarked
+package io.swagger.v3.jaxrs2.resources.jspecify.nullunmarked;
+
+import org.jspecify.annotations.NullUnmarked;
```

Comment:
From my understanding of JSpecify, `@NullUnmarked` is only relevant within a `@NullMarked` package, so you could move this package to `...jspecify.nullmarked.nullunmarked`